pkgs/by-name/om/omake/package.nix

```diff
@@ -1,18 +1,20 @@
 {
   lib,
   stdenv,
-  fetchurl,
+  fetchFromGitHub,
   ocaml,
+  version ? "0.10.7",
```

Comment:
Why doing this?
If you're looking for overriding the version, use the standard override mechanism for derivation.
Also, only changing the version is not enough, the hash needs to be updated too, so I don't see the point in doing this yet.

Reply:
Getting rid of `rec`.

Reply:
Then in that case, why not use let in or simply the finalAttrs pattern ?

Reply:
Thanks for the suggestion. I’m not yet used to `finalAttrs`.

Reply:
It's fine. If you don't like the `finalAttrs`, you could have used a `let...in` too, but honestly, most of the builders now are supporting the `finalAttrs` pattern and I would definitely prioritize it over something else.